lib/pages/OnboardingPage.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:introduction_screen/introduction_screen.dart';
+import 'RegistrationForm.dart';
+
+class OnboardingPage extends StatefulWidget {
+  const OnboardingPage({Key? key}) : super(key: key);
+
+  @override
+  State<OnboardingPage> createState() => _OnboardingPageState();
+}
+
+class _OnboardingPageState extends State<OnboardingPage> {
+  void _onOnboardingEnd(context) {
```

Comment:
I know we had mentioned previously having the ability to go back to the onboarding page through the settings, at which point culminating in the registration page isn't helpful. IDK how viable this is, but can we maybe pre-place the registration page underneath on the stack before opening onboarding when they first log in, and then just remove the onboarding from the stack when it's through?

That way, the settings page is underneath when viewing in isolation and the registration is underneath when entering fresh, and when it removes itself the ideal next place is waiting for them already.

Reply:
I like this idea — going to create a separate issue for it.